### PR TITLE
Add basic Health check

### PR DESCRIPTION
**Pull request by progressions**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -43,6 +43,7 @@ defmodule Todo.Mixfile do
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:gettext, "~> 0.11"},
       {:plug_cowboy, "~> 2.0"},
+      {:poison, "~> 3.1"},
```
Comment (rwalters) [inline mix.exs]: Why did you add Poison back?

Comment (progressions) [reply mix.exs]:
```
[~/projects/elixir/todo-api-ex]> mix phx.server
Compiling 25 files (.ex)
Generated todo app
** (Mix) Could not start application poison: could not find application file: poison.app
```

That's why
